core/backend/src/NavigationRelationship.ts

```diff
@@ -287,3 +287,15 @@ export class SheetOwnsSheetInformationAspect extends ElementOwnsUniqueAspect {
     super(sheetId, relClassName);
   }
 }
+
+/** Relates an [[Element]] to the [[ChannelRootAspect]] that identifies it as a channel root.
+ * @see [[ChannelControl.makeChannelRoot]] to create an instance of this relationship.
+ * @beta
```

Comment:
```suggestion
 * @public
```
The BIS class this represents has been in BisCore since 01.00.10 and your entire API consists solely of a constructor.